server/src/com/cloud/storage/snapshot/SnapshotManagerImpl.java

```diff
@@ -187,6 +193,19 @@
 
     private int _totalRetries;
     private int _pauseInterval;
+    private int maxRetries, retryInterval;
```

Comment:
There is a totalRetries already existing, can you rename the parameters to indicate that there are for snapshot backup